### PR TITLE
feat: Show specific description when no IaC or OSS files found but corresponded scan performed [ROAD-644]

### PR DESCRIPTION
**Pull request by ArtsiomCh**
1. if no file supported by IaC is found, on click on the IaC root node in the tree, the vulnerability description panel should display the following text  (from CLI): `Could not find any valid IaC files`
2. if no file supported by OSS is found, on click on the OSS root node in the tree, the vulnerability description panel should display the following text (from CLI): `Could not detect supported target files in <project-path>.\nPlease see our documentation for supported languages and target files: https://snyk.co/udVgQ and make sure you are in the right directory.`

image

image

### CODE REVIEW

File: src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt [hunk 1]
```diff
@@ -342,3 +344,36 @@ fun wrapWithScrollPane(panel: JPanel): JScrollPane {
     )
     return scrollPane
 }
+
+// "stolen" from https://stackoverflow.com/a/12053940/7577274
```
Comment (bastiandoetsch) [inline src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt:348]: Please have a look at `StringEscapeUtils.escapeHtml`. That should do the job and is part of our dependencies anyway.

Comment (bastiandoetsch) [reply src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt:348]: https://howtodoinjava.com/java/string/escape-html-encode-string/

**Review by bastiandoetsch — APPROVED** [review]
Looks good besides the unneeded `txt2Html` method. Nice!

File: src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt (hunk 1 above)
Comment (ArtsiomCh) [reply src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt:348]: Here we need not only escape html but also replace `\n` and convert html link into real html link (`<href>`)
But good point to delegate that escaping to the library!

**Pull request merged**